demos/common/src/ie_config_helper.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "samples/ie_config_helper.hpp"
+
+void configureInferenceEngine(Core& ie,
+                              std::string& deviceString,
+                              std::string& deviceInfo,
+                              const std::string& lString,
+                              const std::string& cString,
+                              bool pc) {
+    for (char& ch : deviceString) {
+        ch = std::toupper(ch);
```

Comment:
This can be tricky. In case of multiple sticks device names start to contain lower case. I don't know if IE accepts that ids capitalized. It can probably be tested on https://devcloud.intel.com/edge/